### ISSUE TITLE
Provide MO ongoing support

### ISSUE THREAD

**Issue by sychang** [Recidiviz/pulse-dashboard#398]
Jan 24, 2021 update: Remaining burndown tasks are in this sheet: https://docs.google.com/spreadsheets/d/1zIF0L0N1Mv8uHhAZK7XXq1Nwe4-e37nHloseG76axC8/edit#gid=1611380413

[WIP MO ongoing product support strategy doc
](https://docs.google.com/document/d/1HLRRxggyMgl_repc7DV2OU3vLIYM_FkyCnAt4KwPiuw/edit#heading=h.ui202x7eppr3). 

As part of ongoing support and growth of the dashboard, MO has a list of requests that they would like to be implemented long-term. The requests can be roughly categorized into:
1. Building trust in the tool so users have full confidence in the decisions they base off the tool and are willing to integrate it seamlessly into their workflow.
2. Enabling more users: Currently the tool is targeted at middle management and there are a set of features that would enable MO to distribute the tool to district administrators and supervisors, as well as upper-level management.
3. Provide additional interesting statistical breakdowns that can help stakeholders make decisions.

This is the master prioritized list of requests from MO, ranked from top to bottom in terms of priority to them:

1. Individuals revoked (violation record column): Use most severe violation on each report, rather than counting multiple violations on the same report. https://github.com/Recidiviz/recidiviz-data/issues/3353
2. Individuals revoked (recommendation column): Take recommendation on the most recent violation report, including supplementals here (but not for the rest of the page). https://github.com/Recidiviz/recidiviz-data/issues/3358 and #401
3. Deferred prosecution cases should be excluded from the general cases until they are included as a separate case type in the drop down. https://github.com/Recidiviz/recidiviz-data/issues/3352
4. District revocation rates: get rid of inactive districts somehow so that they don’t impact the scale of the graph. #396 
5. For district administrators and unit supervisors, show only the state-wide view and their district. Lock them out of other districts. TODO: 
6. Violation record: Break into stacked bar showing proportion attributable to VR or NOC #385
7. [Replace with re-visualize race and gender charts] Remove comparison between race/gender and risk level. TODO: @sychang file an issue
8. Update rate charts to show exits as denominator (district and risk charts) #395
9. [Blocked on MO] Change revocation cluster lookback period to 9 months (or generally make it shorter). TODO: @sychang file an issue
10. [Blocked] Include all 20+ case types, including things like electronic monitoring, supervision level breakouts, initial assessment phase, deferred prosecution, supervision level 1, level 2, level 3, etc. #397
11. [Blocked] Enable custom date range selection. #335
12. [Blocked] Revocation cluster lookback period: ability to filter by shorter lookback period by 1 mo, 3 mos, 6 mos, 9 mos, 12 mos, and custom date range. TODO: @sychang file an issue
13. [Blocked by scoping] Re-envisioning race and gender charts. TODO: Long-term, needs scoping work.
14. [Blocked on MO] Row in the chart for really bad special conditions violations—defined by set of rules provided by MO.
15. [Blocked on MO] Nest technical row to include breakout by each condition. TODO: Needs to be scoped or punted.
16. [Blocked on scoping] Anomaly detection—follow up discussions about setting the right baseline or threshold. Long-term, needs scoping work.
17. [Blocked on scoping] Find a way to distinguish between people who are unassessed because they’re in the initial days of supervision, and those for whom assessment is overdue. TODO: @sychang file an issue

Other related issues that have come up separately:
18. Capias and CODS reclassification, [3350](https://app.zenhub.com/workspaces/recidiviz-corelantern-5ec36552cdfe7bc58f04bdbe/issues/recidiviz/recidiviz-data/3350)

**Comment by sychang** [comment]
Splitting up this epic into
- Part 1. Things that are unblocked
- Part 2. Things that are blocked

We'll see how this goes...

**Comment by sychang** [comment]
Updated burndown list is here: https://docs.google.com/spreadsheets/d/1zIF0L0N1Mv8uHhAZK7XXq1Nwe4-e37nHloseG76axC8/edit#gid=1611380413
